common/source/docs/common-serial-led-ProfiLED.rst

```diff
@@ -0,0 +1,12 @@
+.. _common-serial-led-ProfiLED:
+
+ProfiLED
+========
+
+.. image:: ../../../images/ProfiLED.jpg
+
+ProfiLED are spaced at 20mm resulting in 50 LED's per meter. The strips can be cut at any point and extended or have a new header added and plugged into a second pair of output pins. ProfiLED's require both clock and data signal. As supplied the clock is the yellow wire and green is data. Both the clock and data pins must be in the same PWM group. It is possible to share a clock pin over multiple strips in the same group. If there are spare pins it may be more convenient to setup two clock pins for that group. ProfiLED's require 5 volts and can draw a significant amount of current if all turned on at once. ProfiLED's default to white until a valid signal is received.
```

Comment:
Pete, I think we need a diagram or two to show exactly how to connect to the autopilot in addition to the words

Reply:
They do come with the correct header on, you just plug them in, I will try and find a picture

Reply:
added a pic of them connected to a cube